Marlin/src/lcd/marlinui.cpp

```diff
@@ -656,21 +660,38 @@ constexpr uint8_t epps = ENCODER_PULSES_PER_STEP;
       LIMIT(new_frm, 10, 999);
 
       if (old_frm != new_frm) {
-        feedrate_percentage = new_frm;
-        encoderPosition = 0;
-        #if BOTH(HAS_BUZZER, BEEP_ON_FEEDRATE_CHANGE)
-          static millis_t next_beep;
-          #ifndef GOT_MS
-            const millis_t ms = millis();
-          #endif
-          if (ELAPSED(ms, next_beep)) {
-            buzz(FEEDRATE_CHANGE_BEEP_DURATION, FEEDRATE_CHANGE_BEEP_FREQUENCY);
-            next_beep = ms + 500UL;
-          }
+        #if ENABLED(ULTIPANEL_FEEDMULTIPLY)
+          planner. = new_frm;
```

Comment:
typo there, should be feedrate_percentage = new_frm; ?

Reply:
Sorry. I can't find the part of code your mean.`
The resulting code is:`
```
     if (old_frm != new_frm) {
        #if ENABLED(ULTIPANEL_FEEDMULTIPLY)
          feedrate_percentage = new_frm;
        #endif
        #if ENABLED(ULTIPANEL_FLOWMULTIPLY)
          planner.flow_percentage[active_extruder] = new_frm;
        #endif
```
And it is correct in my opinion.

Reply:
The pull request contains 3 commits...

Reply:
*_frm means differnt thing in case of differnt define enabed.
Flow or feed rate.
```
      #if ENABLED(ULTIPANEL_FEEDMULTIPLY)
        const int16_t old_frm = feedrate_percentage;
      #endif
      #if ENABLED(ULTIPANEL_FLOWMULTIPLY)
        const int16_t old_frm = planner.flow_percentage[active_extruder];
      #endif
```

Reply:
oh indeed, weird, sorry

Reply:
In any case thank you for attention to my code :)